webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -306,6 +308,19 @@ export const GenericModal: React.FC<Props> = ({
                                 </div>
                             )}
                         </div>
+                        {showCloseButton && (
+                            <button
+                                type='button'
+                                className='close'
+                                onClick={onHideCallback}
+                                aria-label={intl.formatMessage({id: 'generic_modal.close', defaultMessage: 'Close'})}
```

Comment:
I forgot this came from React Bootstrap before. I'm pretty sure that means it wasn't translatable, so nice fix!